packages/frontend/src/components/markdown-container/chakra-ui-renderer.tsx

```diff
@@ -232,91 +274,53 @@ export const ChakraUIRenderer = (customRenderers: Renderers = {}, transformAsset
 
       return (
         <FetchInsightList
-          query={props.query}
+          query={query}
           sort={sort}
-          options={{ ...attributes, dispatchSearch: false }}
+          options={{ ...destringObject(props), dispatchSearch: false }}
           mb="1rem"
         />
       );
     },
-    vegaChart: ({ node, attributes }) => {
-      return <VegaRendererAsync specString={node.config} transformAssetUri={transformAssetUri} {...attributes} />;
-    },
-    video: ({ node, ...props }) => {
+    vegachart: ({ node, config, ...props }) => {
       return (
-        <VideoRenderer
-          url={props.url}
-          mimeType={getMimeForFileName(props.url)}
+        <VegaRendererAsync
+          key={hashCode(config)}
+          specString={config}
           transformAssetUri={transformAssetUri}
-          {...node.attributes}
+          {...props}
         />
       );
     },
-    xkcdChart: ({ node, attributes }) => {
-      return <XkcdChartRendererAsync type={node.xkcdType} configString={node.config} {...attributes} />;
-    },
-    footnoteReference: ({ node, ...props }) => {
```

Comment:
`footnoteReference` and `footnoteDefinition` aren't exposed anymore, `remark-gfm` converts directly to HTML now.

Reply:
Oh nice! So that's why we don't need the remarkFootnotes plugin anymore?